cmd/tailscale/cli/web.go

```diff
@@ -83,6 +85,35 @@ var webArgs struct {
 	cgi    bool
 }
 
+func tlsCertFromEnvironment(listen string) (*http.Server, error) {
+	crt := os.Getenv("TLS_CRT")
```

Comment:
these names would make me assume these are paths to files on disks, not the PEM contents themselves.

Reply:
Added "_PEM" to environment variable names.